drivers/nrf9160_gps/nrf9160_gps.c

```diff
@@ -158,6 +179,27 @@ static void gps_thread(int dev_ptr)
 
 		switch (raw_gps_data.data_id) {
 		case NRF_GNSS_PVT_DATA_ID:
+			if (gps_is_blocked(&raw_gps_data.pvt)) {
+				if (operation_blocked) {
+					/* Avoid spamming the logs. */
+					continue;
+				}
+
+				LOG_INF("Waiting for time window to operate");
+				LOG_DBG("PSM or eDRX or DRX needs to be enabled for the GPS to operate");
```

Comment:
I think this is only true in the case you want the modem to automatically switch between LTE and GPS. In the case you want to do GPS only or you want the application to control the switching between LTE and GPS this is not true.

Reply:
Good catch, thanks. Will fix that.